rust/examples/honeycomb.rs

```diff
@@ -0,0 +1,80 @@
+use dylibso_observe_sdk::adapter::honeycomb::{HoneycombAdapter, HoneycombConfig};
+use dylibso_observe_sdk::adapter::otel_formatter::{Attribute, Value};
```

Comment:
I think it would be good to re-export these `Attribute` and `Value` types from the `dylibso_observe_sdk::adapter::honeycomb` module. Then we can skip the `otel_formatter` as an explicit import.

Reply:
(i pushed up a change to do this!)